### PR TITLE
Fix: Box Plus channels

### LINKED ISSUE #3237: Fix: Box Hits Uk
**_Channel Name:_** Box Hits UK
**_Broken Link (from playlist):_** http://csm-e.tm.yospace.com/csm/extlive/boxplus01,boxhits-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/boxhits/
**_Possible Replacement (optional):_** n/a
**_Notes (optional):_**

### LINKED ISSUE #3238: Fix: The Box UK
**_Channel Name:_** The Box UK
**_Broken Link (from playlist):_** http://csm-e.tm.yospace.com/csm/extlive/boxplus01,thebox-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/thebox/
**_Possible Replacement (optional):_** n/a
**_Notes (optional):_**

### LINKED ISSUE #3239: Fix: Kerrang UK
**_Channel Name:_** Kerrang UK
**_Broken Link (from playlist):_** http://csm-e.tm.yospace.com/csm/extlive/boxplus01,kerrang-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/kerrang/
**_Possible Replacement (optional):_** n/a
**_Notes (optional):_**

### PR DESCRIPTION
**Pull request by Dum4G**
Closes https://github.com/iptv-org/iptv/issues/3239
Closes https://github.com/iptv-org/iptv/issues/3238
Closes https://github.com/iptv-org/iptv/issues/3237

### CODE REVIEW

File: channels/uk.m3u [hunk 1]
```diff
@@ -152,8 +152,10 @@ http://91.218.60.3:8001/play/a00t
...
 #EXTINF:-1 tvg-id="BeanoTV.uk" tvg-name="Beano TV" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/XvBi1Ou.png" group-title="",Beano TV (720p)
 https://beanostudios-beanotv-1-gb.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="BoxHits.uk" tvg-name="Box Hits" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="Music",Box Hits (576p)
-http://csm-e.tm.yospace.com/csm/extlive/boxplus01,boxhits-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/boxhits/
+#EXTINF:-1 tvg-id="BoxHits.uk" tvg-name="Box Hits" tvg-country="UK" tvg-language="English" tvg-logo="https://tvepg.eu/img/united_kingdom/logo/box_hits.png" group-title="Music",Box Hits (576p)
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,boxhits-alldev.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/boxhits/
+#EXTINF:-1 tvg-id="BoxHits.uk" tvg-name="Box Hits" tvg-country="UK" tvg-language="English" tvg-logo="https://tvepg.eu/img/united_kingdom/logo/box_hits.png" group-title="Music",Box Hits (576p) [Geo-blocked]
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,boxhits-alldev.m3u8?yo.up=https://boxtv.secure.footprint.net/boxhits/
```
Comment (LaneSh4d0w) [inline channels/uk.m3u:158]: It's difficult to get the stream from this link.

File: channels/uk.m3u [hunk 2]
```diff
@@ -286,7 +288,9 @@ http://188.40.68.167/russia/jimjam/playlist.m3u8
 #EXTINF:-1 tvg-id="KanshiTV.uk" tvg-name="Kanshi TV" tvg-country="UK" tvg-language="Hindi" tvg-logo="https://i.imgur.com/jxf9gQd.jpg" group-title="",Kanshi TV
 https://live.kanshitv.co.uk/mobile/kanshitvkey.m3u8
 #EXTINF:-1 tvg-id="Kerrang.uk" tvg-name="Kerrang" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/Nn2VfkP.png" group-title="",Kerrang (576p)
-http://csm-e.tm.yospace.com/csm/extlive/boxplus01,kerrang-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/kerrang/
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,kerrang-alldev.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/kerrang/
+#EXTINF:-1 tvg-id="Kerrang.uk" tvg-name="Kerrang" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/Nn2VfkP.png" group-title="",Kerrang (576p) [Geo-blocked]
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,kerrang-alldev.m3u8?yo.up=https://boxtv.secure.footprint.net/kerrang/
```
Comment (LaneSh4d0w) [inline channels/uk.m3u:293]: It's difficult to get the stream from this link.

File: channels/uk.m3u [hunk 3]
```diff
@@ -425,8 +429,10 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-rokuteletubbies/CDN/playlist.m3u8
...
 https://tennischannel-int-samsunguk.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="TheBoxUK.uk" tvg-name="The Box UK" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="Music",The Box UK (576p)
-http://csm-e.tm.yospace.com/csm/extlive/boxplus01,thebox-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/thebox/
+#EXTINF:-1 tvg-id="TheBoxUK.uk" tvg-name="The Box UK" tvg-country="UK" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/1/16/The_Box_2013.png" group-title="Music",The Box UK (576p)
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,thebox-alldev.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/thebox/
+#EXTINF:-1 tvg-id="TheBoxUK.uk" tvg-name="The Box UK" tvg-country="UK" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/1/16/The_Box_2013.png" group-title="Music",The Box UK (576p) [Geo-blocked]
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,thebox-alldev.m3u8?yo.up=https://boxtv.secure.footprint.net/thebox/
```
Comment (LaneSh4d0w) [inline channels/uk.m3u:435]: It's difficult to get the stream from this link.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

**Comment by LaneSh4d0w** [comment]
The other 3 above the ones I've mentioned works pretty fine.

File: channels/uk.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/uk.m3u:158]: Have you tried a UK VPN? I think MapGuy did

Comment (LaneSh4d0w) [reply channels/uk.m3u:158]: On the 3 links above, I didn't need to. It worked directly. I'll try with a UK VPN.

Comment (LaneSh4d0w) [reply channels/uk.m3u:158]: Using UK Tor link, it's buffering but doesn't give me anything else.
image

Comment (Dum4G) [reply channels/uk.m3u:158]: image

**Review by MapGuy11 — APPROVED** [review]

File: channels/uk.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/uk.m3u:158]: Well if that works, that works.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**